webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -704,14 +704,36 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                         {% if frame.truncated %}
                           <tr><td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td></tr>
                         {% else %}
+                          {% if frame.inlines %}
+                            {% for inline in frame.inlines %}
+                              <tr class="{% if frame.missing_symbols %}missingsymbols{% endif %}">
+                                <td>
+                                  {% if frame.missing_symbols %}
+                                    <span class="row-notice" title="missing symbol">&Oslash;</span>
+                                  {% endif %}
+                                  {{ frame.frame }}
+                                </td>
+                                <td>{{ frame.module }}</td>
+                                <td title="{{ inline.function }}">{{ inline.function }}</td>
+                                <td>
+                                  {% if inline.source_link %}
+                                    <a href="{{ inline.source_link }}">{{ inline.file }}:{{ inline.line }}</a>
+                                  {% else %}
+                                    {% if inline.file %}{{ inline.file }}:{{ inline.line }}{% endif %}
+                                  {% endif %}
+                                </td>
+                                <td>inlined</td>
+                              </tr>
+                            {% endfor %}
+                          {% endif %}
```

Comment:
Per these docs, the inline functions should get printed _before_ the frame:

https://github.com/rust-minidump/rust-minidump/blame/188ec7e4bab55a66d76c7171de149a37381d6ca6/minidump-processor/json-schema.md#L309-L317

This change fixes that.